packages/ui-components/style/styling.css

```diff
@@ -46,6 +46,11 @@ input.jp-mod-styled:focus {
   box-shadow: inset 0 0 4px var(--md-blue-300);
 }
 
+input[type='checkbox'].jp-mod-styled:focus {
```

Comment:
```suggestion
input[type='checkbox'].jp-mod-styled:focus-visible {
```

Reply:
It feels like we should only show the indicator when browser believes that the focus is applicable.